foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/http/HttpUtils.java

```diff
@@ -94,6 +97,14 @@ public static String encodePathParam(String pathParam) {
     return UrlEscapers.urlPathSegmentEscaper().escape(pathParam);
   }
 
+  public static String decodePathParam(String pathParam) throws UnsupportedEncodingException {
+    String res = uriDecodePath(pathParam);
+    if (StringUtils.isEmpty(res)) {
+      return URLDecoder.decode(pathParam, "UTF-8");
```

Comment:
Is ``` URLDecoder.decode(pathParam, "UTF-8");``` is enough?  I think it is enough to use URLEncoder and URLDecoder to encode/decode path segments, and use URI to encode/decode path.

Reply:
`URLDecoder.decode(pathParam, "UTF-8")  `can not handle the '+' .
there is a ut show that : org.apache.servicecomb.common.rest.codec.param.TestPathProcessor#testGetPlus.